cmd/test/main.go

```diff
@@ -515,3 +519,18 @@ func wereViolationsFound(validationManager *ValidationManager, results *evaluati
 		return false
 	}
 }
+
+func saveDefaultRulesAsFile(ctx *TestCommandContext, preRunPolicies string) {
+	if preRunPolicies == "" {
```

Comment:
don't we always get preRunPolicies? when would it be empty?

Reply:
I don't know but since the cli-service is a different service and could eventually change, I think we cannot depend on it to always have a value